ntbs-service/Services/UserService.cs

```diff
@@ -18,6 +18,8 @@ public interface IUserService
         Task<UserPermissionsFilter> GetUserPermissionsFilterAsync(ClaimsPrincipal user);
         Task<TBService> GetDefaultTbService(ClaimsPrincipal user);
         Task<IEnumerable<TBService>> GetTbServicesAsync(ClaimsPrincipal user);
```

Comment:
Do we ever use the TbServices as a whole entity from this? Or should we potentially move it to return strings too?

Reply:
yes, this is used on different methods, I can write a new method, but seems unnecessary